docs/extension/guides/create-backup.md

```diff
@@ -8,31 +8,32 @@ description: 'Extension Support: Create Backup'
 
 If you lost access to the Universal Profile Browser Extension, there are multiple ways to secure your Universal Profile :
 
-- **2FA**: You can set up 2FA for the Universal Profile on [UniversalProfile.cloud](https://universalprofile.cloud/). 2FA is the recommended way of backing up a Universal Profile and will add a new recovery controller to the profile, secured on LUKSO's end. This controller has permission to add new controllers if access to an old extension is lost. For security reasons, users must provide an authentication code from a mobile device and approve via email before a new controller can be added using LUKSO's recovery service.
+- **2FA Recovery**: You can set up 2FA for the Universal Profile on [my.universalProfile.cloud](https://mw.universalprofile.cloud/). 2FA is the recommended way of backing up a Universal Profile and will add a new recovery controller to the profile, secured on LUKSO's end. The recovery controller has permissions to add additional controllers if access to an old extension is lost. For security reasons, users must provide an authentication code from a mobile device and approve via email before a new controller can be added using LUKSO's recovery service.
+- **Wallet Recovery**: You can give recovery rights to an other blockchain account or controller on [my.universalProfile.cloud](https://mw.universalprofile.cloud/). This blockchain account then gains permissions to add additional controllers if access to an old extension is lost. Wallet recovery can also be used if you want to regain access to your Universal Profile using the revealed private key of the profile's extension controller.
```

Comment:
```suggestion
- **Wallet Recovery**: You can give recovery rights to another blockchain account or controller on [my.universalprofile.cloud](https://my.universalprofile.cloud/). This blockchain account then gains permissions to add additional controllers if access to an old extension is lost. Wallet recovery can also be used if you want to regain access to your Universal Profile using the private key of the profile's extension controller.
```

I'm not sure what "revealed private key" means. I think "private key" is enough.